src/ring_theory/algebra.lean

```diff
@@ -357,6 +358,15 @@ variables [algebra R A₁] [algebra R A₂] [algebra R A₃]
 
 instance : has_coe_to_fun (A₁ ≃ₐ[R] A₂) := ⟨_, alg_equiv.to_fun⟩
 
+@[ext]
+lemma ext {f g : A₁ ≃ₐ[R] A₂} (h : ∀ a, f a = g a) : f = g :=
+begin
+  have h₁ : f.to_equiv = g.to_equiv := equiv.ext h,
```

Comment:
Shouldn't there be a `to_(ring_)equiv_inj` lemma? I guess that's something that will occasionally be useful in proofs as well.

Reply:
Okay. I added two lemmas along these lines.